Sources/Plot/API/HTMLComponents.swift

```diff
@@ -850,3 +852,24 @@ public struct TextArea: InputComponent {
         )
     }
 }
+
+/// Component that represents a datetime instance
+public struct Time: Component {
+    /// A closure that provides the components that the row should contain.
```

Comment:
I think this is a copy/paste or duplicate file error. This should probably say:
`A closure that provides the contained content.`

Or something like that. :-)

Reply:
Ah yeh that's my bad, I'll fix that quickly

Reply:
Should be fixed in the latest commit: af0927f1409e5ceae4071d3d0172c3ab86c4c3f0